index.html

```diff
@@ -466,347 +466,365 @@ <h3>Ethereum EIP712 Signature 2021</h3>
-                {
-                  "name": "child",
-                  "type": "Person"
+                  "type": "Name"
                 }
               ],
-              "EIP712Domain": [
+              "Document": [
                 {
-                  "name": "name",
-                  "type": "string"
+                  "name": "@context",
+                  "type": "string[]"
                 },
                 {
-                  "name": "version",
+                  "name": "@type",
                   "type": "string"
                 },
                 {
-                  "name": "chainId",
-                  "type": "uint256"
+                  "name": "data",
+                  "type": "Data"
                 },
                 {
-                  "name": "salt",
-                  "type": "bytes32"
-                }
-              ],
-              "Person": [
-                {
-                  "name": "type",
-                  "type": "string"
-                },
-                {
-                  "name": "name",
+                  "name": "telephone",
                   "type": "string"
                 }
               ],
-              "Proof": [
-                {
-                  "name": "verificationMethod",
-                  "type": "string"
-                },
+              "Job": [
                 {
-                  "name": "created",
+                  "name": "employer",
                   "type": "string"
                 },
                 {
-                  "name": "proofPurpose",
-                  "type": "string"
-                },
-                {
-                  "name": "type",
+                  "name": "jobTitle",
                   "type": "string"
                 }
               ],
-              "VerifiableCredential": [
-                {
-                  "name": "@context",
-                  "type": "string[]"
-                },
-                {
-                  "name": "type",
-                  "type": "string[]"
-                },
-                {
-                  "name": "id",
-                  "type": "string"
-                },
+              "Name": [
                 {
-                  "name": "issuer",
+                  "name": "firstName",
                   "type": "string"
                 },
                 {
-                  "name": "issuanceDate",
+                  "name": "lastName",
                   "type": "string"
-                },
-                {
-                  "name": "credentialSubject",
-                  "type": "CredentialSubject"
-                },
-                {
-                  "name": "credentialSchema",
-                  "type": "CredentialSchema"
-                },
-                {
-                  "name": "proof",
-                  "type": "Proof"
                 }
               ]
             },
-            "primaryType": "VerifiableCredential"
+            "domain": {},
```

Comment:
Shouldn't this `domain` object have one or more of the required fields, such `name` or `version`?

<https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator>:

> the type of eip712Domain is a struct named EIP712Domain with one or more of the below fields.

Reply:
added `name` to all domain properties for test vectors